### PR TITLE
Add: commit on init with tram-one

### PR DESCRIPTION
**Pull request by abhishekr700**
Solution to issue https://github.com/Tram-One/tram-one-express/issues/57 
After the run of `generator.js` , the project dir will be initialized with a git repo and first commit will be done.
Signed-off-by: Abhishek Ranjan <abhishekr700@gmail.com>

### CODE REVIEW

File: generator.js [hunk 1]
```diff
@@ -43,11 +44,21 @@ const processFile = (file, currentPath) => {
...
+
 console.log(`Creating ${projectPath} `)
 console.log('Copying over project files')
 processFile('', filePath)
 console.log('Installing NPM Depenedencies')
 execSync('npm install', {cwd: projectPath, stdio: 'inherit'})
+console.log('Initializing a git repository')
+console.log('Making the initial commit !')
+const simplegit = git(projectPath)
+simplegit.init().add('./*').commit('Init Tram-One-App !', (e, d) => {
```
Comment (JRJurman) [inline generator.js]: * not `add('.')`? 
* if we could make this string instead `'Initial commit from Tram-One Express'`, this follows the commit message that create-react-app uses
* please use full variables here, not `e` and `d` (we don't have to worry about bundle size here 😄)

File: generator.js [hunk 2]
```diff
@@ -43,11 +44,21 @@ const processFile = (file, currentPath) => {
...
+  if (e) console.log(e)
+  else {
+    console.log(d)
+  }
```
Comment (JRJurman) [inline generator.js]: what does the logging usually show? potentially we should have our own strings here depending on what the messaging usually shows. 

If nothing else, this logic could probably be changed to
```
console.log(e || d)
```

File: generator.js [hunk 3]
```diff
@@ -43,11 +44,21 @@ const processFile = (file, currentPath) => {
...
 const filePath = path.join(__dirname, 'template')
 const projectPath = path.join(process.cwd(), appTitle)
+
 console.log(`Creating ${projectPath} `)
 console.log('Copying over project files')
 processFile('', filePath)
 console.log('Installing NPM Depenedencies')
 execSync('npm install', {cwd: projectPath, stdio: 'inherit'})
+console.log('Initializing a git repository')
+console.log('Making the initial commit !')
```
Comment (JRJurman) [inline generator.js]: To be consistent with the other messaging just remove the `!` here. `'Making the initial commit'` is fine 👍

Comment (abhishekr700) [reply generator.js]: Cool

File: generator.js (hunk 2 above)
Comment (JRJurman) [reply generator.js]: we should avoid logging the data here. Instead we should just log either `Successfully created commit` or `Failed to create commit`. 

If we fail, that's fine, and I'm not too passionate about doing detailed logging here.

**Comment by JRJurman** [comment]
@abhishekr700 looks good! Thanks for making the updates 😄 
Can we get a screenshot or the output of `git log` in the summary? 
I just want to see that this change works as expected 👍

**Comment by abhishekr700** [comment]
Sure, here you go ! 💯 
Screenshot (77)

**Pull request merged**